01-data-structures/03-linked-lists/linked_list_spec.rb

```diff
@@ -89,4 +89,41 @@
       expect(llist.head).to eq nil
     end
   end
-end
\ No newline at end of file
+
+  describe "#locate_node_by_index" do
+    before do
```

Comment:
you should text each one where the list is empty, where there's 1 node, and where there are many nodes for every function